README.md

```diff
@@ -1,4 +1,6 @@
 # OpenShift V3 Plugin for Jenkins
+[![Build Status](https://ci.openshift.redhat.com/jenkins/buildStatus/icon?job=openshift-pipeline-plugin)](https://ci.openshift.redhat.com/jenkins/job/openshift-pipeline-plugin/) [![Build Status](https://travis-ci.org/openshift/jenkins-plugin.svg?branch=master)](https://travis-ci.org/openshift/jenkins-plugin)
```

Comment:
hey @PI-Victor - thanks for the space cleanup!

On pointing to the ci.openshift build status, we only use that job for building pre-release versions of the plugin, and it only generates builds manually.  It doesn't monitor commit activity.  In fact, @mfojtik originally created it so he wouldn't have to build the plugin on his local env's ;-) but I've re-purposed it to provide preliminary patches to users when providing them fixes or feature enhancements.

Let's *NOT* add a link to this in the README.